apps/site/assets/css/_grid-button.scss

```diff
@@ -9,6 +9,54 @@
   margin: $base-spacing * -.25;
 }
 
+.c-grid-buttons-right {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+  align-items: flex-end;
+  margin-top: 1rem;
+  @include media-breakpoint-up(sm) {
+    border-bottom: 2px solid $gray-lightest;
+    padding-bottom: 10px;
+  }
+}
+
+.c-location-buttons-container {
+  display: flex;
+  flex-grow: 2;
+  @include media-breakpoint-down(xs) {
+    flex-wrap: wrap;
+  }
+}
+
+.c-location-header {
+  flex-grow: 2;
+  @include media-breakpoint-down(sm) {
+    flex-basis: 100%;
+  }
+}
```

Comment:
Having both of these set to `flex-grow: 2` encourages these to take up the same width but on larger breakpoints we can let the buttons take up more width. Might need adjustment to prevent wrapping like this
<img width="670" alt="image" src="https://user-images.githubusercontent.com/2136286/169856782-0276ce76-0c38-461e-86ee-19d1a4e76314.png">